Uebungsaufgaben/bibliothek2/src/Renderer.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+
+class Renderer
+{
+    /** @var XSLTProcessor  */
+    private $proc;
+
+    public function __construct(XSLTProcessor $proc)
+    {
+        $this->proc = $proc;
+    }
+
+    public function render(DOMDocument $dom): void
+    {
+        echo $this->proc->transformToXml($dom);
```

Comment:
return statt echo. der aufrufer kann enscheiden ob er es ausgeben will oder in eine datei speichern oder was auch immer